docs/data-sources-api/methods/r_uploadDataDataSources.md

```diff
@@ -31,7 +31,7 @@ Uploads data to a data source.
 
 ## Response
 
-Boolean, true on success, otherwise false if no error occurs.
+Boolean, true on success, otherwise false (one case is if Data Sources is internally not able to write on FTP it will return false ).
```

Comment:
I like that this calls out the possible error cases. What would you think of the following wording inside the parens. 
(one case where false is returned is if Data Sources is internally not able to write the received file)

Reply:
yep let me change it